### PR TITLE
NAS-110119 / 21.06 / Code names are displayed in cloud credentials list

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

File: src/app/pages/credentials/backup-credentials/backup-credentials.component.ts [hunk 1]
```diff
@@ -108,6 +114,20 @@ export class BackupCredentialsComponent implements OnInit, OnDestroy {
     ];
   }
 
+  cloudCredentialsDataSourceHelper(res) {
+    if (this.providers) {
+      res = res.map(item => {
```
Comment (denysbutenko) [inline src/app/pages/credentials/backup-credentials/backup-credentials.component.ts:119]: can we do `return res.map()`?

File: src/app/pages/credentials/backup-credentials/backup-credentials.component.ts [hunk 2]
```diff
@@ -32,14 +32,19 @@ export class BackupCredentialsComponent implements OnInit, OnDestroy {
...
   ngOnInit(): void {
-    this.getCards();
-    this.refreshTable = this.modalService.refreshTable$.subscribe(() => {
-      this.getCards();
-    })
-    this.refreshForms();
-    this.refreshForm = this.modalService.refreshForm$.subscribe(() => {
-      this.refreshForms();
-    });
+    this.cloudCredentialsService.getProviders().subscribe(
```
Comment (denysbutenko) [inline src/app/pages/credentials/backup-credentials/backup-credentials.component.ts:35]: we may need to unsubscribe on destroy, isn't it?

**Comment by AlexKarpov98** [comment]
@denysbutenko Fixed.

File: src/app/pages/credentials/backup-credentials/backup-credentials.component.ts [hunk 3]
```diff
@@ -115,17 +116,15 @@ export class BackupCredentialsComponent implements OnInit, OnDestroy {
...
           item.provider = credentialProvider.title;
         }
-        return item;
-      });
-    }
-
-    return res;
+      }
+      return item;
+    });;
```
Comment (denysbutenko) [inline src/app/pages/credentials/backup-credentials/backup-credentials.component.ts:127]: doubled semicolon

**Comment by AlexKarpov98** [comment]
@denysbutenko Fixed.

**Review by denysbutenko — APPROVED** [review]
image
Looks fine :)

**Pull request merged**